src/components/dashboard/FaceVerification/sdk/UICustomization.css

```diff
@@ -20,6 +20,43 @@
   height: 134px;
 }
 
+#zoom-upload-progress-message, #zoom-session-result-animation-success-message {
+  font-weight: 500 !important;
+}
+
+#zoom-upload-progress-wrapper .loading::before,
+#zoom-upload-progress-wrapper .loading::after {
+  display: inline-block;
```

Comment:
1. We could use just one pseudo element - just add the line break 0x0a ASCII  character ('\A' or '\a' or '\00000a') and set white-space: pre to cast it as plain text so newline character will be processed like `<br>`

2. Why do we need transform ? Couldn't we make the pesudo element absolute and position it via percentage top/left & width ?

I've made a codepen, please check it: https://codepen.io/serdiukov-o-nordwhale/pen/RwaWpJp

Reply:
1. I think we need to leave the implementation with two pseudo-selectors as the style for these strings is different so we need to style them differently.
2. Yeah, I will fix it - see new commit